"Could not find com.google.firebase:firebase-common" when build command is executed

I integrated `firebase-app`, `firebase-firestore` and `firebase-auth` in my KMM project but when I launch a build command, I got this error:

```bash
FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':sample-app:lintReportDebug'.
> Could not resolve all task dependencies for configuration ':sample-app:debugAndroidTestCompileClasspath'.
   > Could not find com.google.firebase:firebase-common:.
     Required by:
         project :sample-app > project :openfeedback-m2 > project :openfeedback > dev.gitlive:firebase-app:1.6.2 > dev.gitlive:firebase-app-android-debug:1.6.2
```

The content of my `build.gradle.kts` is pretty simple:

```kotlin
    sourceSets {
        val commonMain by getting {
            dependencies {
                api(libs.kotlin.coroutines.core)
                api("org.jetbrains.kotlinx:kotlinx-serialization-json:1.4.1")
                api("dev.gitlive:firebase-app:1.6.2")
                implementation("dev.gitlive:firebase-firestore:1.6.2")
                implementation("dev.gitlive:firebase-auth:1.6.2")
                implementation("org.jetbrains.kotlinx:kotlinx-datetime:0.4.0")
            }
        }
        val androidMain by getting
        if (OperatingSystem.current().isMacOsX) {
            val iosX64Main by getting
            val iosArm64Main by getting
            val iosSimulatorArm64Main by getting
            val iosMain by creating {
                dependsOn(commonMain)
                iosX64Main.dependsOn(this)
                iosArm64Main.dependsOn(this)
                iosSimulatorArm64Main.dependsOn(this)
            }
        }
    }
```

If you want to have more context, you can find my branch with `firebase-kotlin-sdk` configured here: https://github.com/GerardPaligot/openfeedback-android-sdk/tree/refactor/ios/firestore

Thanks in advance!

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -126,7 +126,10 @@ subprojects {
         dependencies {
             "commonMainImplementation"(libs.kotlinx.coroutines.core)
             "androidMainImplementation"(libs.kotlinx.coroutines.play.services)
-            "androidMainImplementation"(platform(libs.firebase.bom))
+            // api, not implementation: the Firebase artifacts are declared with `api` and without a
+            // version, so the BoM has to reach the api variant as well or a consumer resolving the
+            // compile classpath has nothing to supply the version from.
+            "androidMainApi"(platform(libs.firebase.bom))
             "commonTestImplementation"(kotlin("test-common"))
             "commonTestImplementation"(kotlin("test-annotations-common"))
             if (this@afterEvaluate.name != "firebase-crashlytics") {

